client/ayon_core/hosts/houdini/plugins/load/load_asset_lop.py

```diff
@@ -0,0 +1,52 @@
+from ayon_core.pipeline import load
+from ayon_core.hosts.houdini.api.lib import find_active_network
+
+import hou
+
+
+class LOPLoadAssetLoader(load.LoaderPlugin):
+    """Load reference/payload into Solaris using AYON `lop_import` LOP"""
+
+    product_types = {"*"}
+    label = "Load Asset (LOPs)"
+    representations = ["usd", "abc", "usda", "usdc"]
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name=None, namespace=None, data=None):
+
+        # Define node name
+        namespace = namespace if namespace else context["folder"]["name"]
+        node_name = "{}_{}".format(namespace, name) if namespace else name
+
+        # Create node
+        network = find_active_network(
+            category=hou.lopNodeTypeCategory(),
+            default="/stage"
+        )
+        node = network.createNode("ayon::lop_import", node_name=node_name)
+        node.moveToGoodPosition()
+
+        # Set representation id
+        parm = node.parm("representation")
+        parm.set(context["representation"]["id"])
```

Comment:
It would be cool if, in the future, we have the representations as a dropdown, too. 
However, I believe it is best to change this in the HDA. 

Here is an option for doing it as a spare Parm. I didn't do much research for implementation yet (just as a warning)
``` 
dropdown_parm = hou.MenuParmTemplate("representations", "representations", menu_items=self.representations)
if node.parmTemplateGroup() is None:
    node.setParmTemplateGroup(hou.ParmTemplateGroup())
node.addSpareParmTuple(dropdown_parm)
```

Reply:
@MustafaJafar just wondering if this still on your radar to look into potentially?

Reply:
Hey, Some updates. 
I added few dropdown menus in this PR to make it easier for users to select the product to load.
https://github.com/BigRoy/ayon-core/pull/3

> the representations as a dropdown, too.

we can do the same for ~~`representations`~~ `product type` dropdown. (I think it should be renamed as mentioned in https://github.com/BigRoy/ayon-core/pull/3#issuecomment-2077866687 )

This code should update the menu entries. 
I don't know if there's a prettier version of it.

```python
import ayon_api
from ayon_core.pipeline import registered_host


def get_available_products_types():
    host = registered_host()
    current_context = host.get_current_context()
    project_name = current_context["project_name"]
    folder_path = current_context["folder_path"]        
    
    id_only = ["id"]
    folder_entity = ayon_api.get_folder_by_path(project_name,
                                                folder_path,
                                                fields=id_only)
    my_folder_id = folder_entity["id"]
    
    my_products = ayon_api.get_products(
        project_name,
        folder_ids = [my_folder_id],
        
    )
    
    available_types = [product["productType"] for product in my_products if product["productType"] != "workfile"]
    
    available_types = list(dict.fromkeys(available_types))
    
    result = []

    for i in available_types:
        result.append(i)
        result.append(i)

    return(result)
```

Reply:
Should I mark this resolved in favor of moving it to a follow-up PR?